e2e_samples/fabric_dataops_sample/README.md

```diff
@@ -40,6 +40,14 @@ At a high level, the deployment sequence proceeds as follows:
 
 Please follow the [How to use the sample](#how-to-use-the-sample) section for detailed instructions.
 
+### Continuous Integration and Continuous Delivery (CI/CD)
```

Comment:
Shall we move it after "Deployed Resources"? Somehow the ordering of headers feels off.

Reply:
I've rearranged a few things. I think this flow is a bit better and a bit more aligned with our existing databricks sample readme.